library/lib.sh

```diff
@@ -120,7 +128,10 @@ rolesHandleVault() {
         fi
     fi
     rlLogInfo "Including vault variables in $playbook_file"
-    export ANSIBLE_VAULT_PASSWORD_FILE="$vault_pwd_file"
+    ansible_env_arg="ANSIBLE_VAULT_PASSWORD_FILE=$vault_pwd_file"
+    if [[ ! $ANSIBLE_ENVS =~ ANSIBLE_VAULT_PASSWORD_FILE ]]; then
```

Comment:
you could also consider making `ANSIBLE_ENVS` a bash array or associative array - the latter is preferred for ease of adding new elements:
```bash
declare -A ANSIBLE_ENVS
...
  if [ -z "${ANSIBLE_ENVS[ANSIBLE_VAULT_PASSWORD_FILE]}" ]; then
    ANSIBLE_ENVS[ANSIBLE_VAULT_PASSWORD_FILE]="$vault_pwd_file"
  fi
```
but then printing it in the format you desire will require a shell function:
```bash
printarr() { declare -n __p="$1"; for k in "${!__p[@]}"; do printf "%s=%s " "$k"  "${__p[$k]@Q}" ; done ; echo ; }
...
cmd="$(printarr ANSIBLE_ENVS) ansible-playbook -i $inventory $skip_tags $limit $tests_path$test_playbook -vv"
```